doc/advanced-topics.md

```diff
@@ -368,6 +368,28 @@ This is a flag that can be enabled by activating the parameter `appendMode` in t
 activated, the update requests to the Context Broker will be performed always with APPEND type, instead of the default
 UPDATE. This have implications in the use of attributes with Context Providers, so this flag should be used with care.
 
+### Differences between `autoprovision`, `explicitAttrs` and `appendMode`
```

Comment:
Should be included in the ToC at the start of the file.

(Btw, it seems "Configuring operation to persist the data in Context Broker (appendMode)" entry is duplicated in the ToC. It would be great to fix that also).

Reply:
Fixed in https://github.com/telefonicaid/iotagent-node-lib/pull/1306/commits/1fe701f0c76398c336f88eabf433ae1dcdb7d31b